src/emulator/hub.ts

```diff
@@ -98,6 +98,11 @@ export class EmulatorHub extends ExpressBasedEmulator {
     });
 
     app.post(EmulatorHub.PATH_EXPORT, async (req, res) => {
+      if (req.headers.origin) {
+        res.status(401).json({
```

Comment:
This should be 403 since additional authentication (like a header wouldn't help)

Reply:
```suggestion
        res.status(403).json({
```